Kubernetes/audit-log/_meta/smart-descriptions.json

```diff
@@ -0,0 +1,159 @@
+[
+    {
+        "value": "{user.name} executed a command in {kubernetes.resource}/{kubernetes.object.name} in namespace {kubernetes.namespace} (from {source.ip})",
+        "conditions": [
+            {
+                "field": "user.name"
+            },
+            {
+                "field": "event.action",
+                "value": "create"
+            },
+            {
+                "field": "kubernetes.subresource",
+                "value": "exec"
+            },
+            {
+                "field": "kubernetes.resource"
+            },
+            {
+                "field": "kubernetes.object.name"
+            },
+            {
+                "field": "kubernetes.namespace"
+            }
+        ],
+        "relationships": []
```

Comment:
Most relationships in this file are empty, I think they could be filled easily (like line 71 is)